psydac/api/tests/navier_stockes.py

```diff
@@ -0,0 +1,280 @@
+# -*- coding: UTF-8 -*-
```

Comment:
Please change the file name: "Navier Stokes" has no "c"

Reply:
done